Admin/AdminInterface.php

```diff
@@ -662,6 +662,13 @@ public function delete($object);
      *
      * @return mixed
      */
+    public function preValidate($object);
```

Comment:
This cannot be added for the 2.X released (can you just comment these line and add a todo)